src/server/controller/metadata/index.ts

```diff
@@ -22,6 +23,7 @@ import { updateTableSection } from './updateTableSection'
 export const MetadataController = {
   // export
   exportAll,
+  importAll,
```

Comment:
@sorja this pr looks good to me.
However, originally I had in mind to import metadata and areas (and in the future data) through their respective controllers. however, looking at the current implementation, it's a generic table export/import. I think in a next pr we should move import and export methods.
Should we add under db a DBService with two methods importTables and exportTables ?
Any other suggestions ?